app/pages/_app.tsx

```diff
@@ -5,6 +5,7 @@ import { getClientEnvironment } from "../lib/relay/client";
 import "normalize.css";
 import BCGovTypography from "components/BCGovTypography";
 import SessionExpiryHandler from "components/SessionExpiryHandler";
+import "semantic-ui-css/semantic.min.css";
```

Comment:
that's a lot of CSS to import in every page for just one component

Reply:
Something like MUI's [Autocomplete](https://mui.com/components/autocomplete/) might be a better option if we want to bring in a single, customizable, component

Reply:
We might get away with just a few files: https://github.com/Semantic-Org/Semantic-UI-React/issues/2029#issuecomment-326455590

Reply:
I'm not super familiar with SemanticUI, but it looks much easier to make a theme for MUI than SemanticUI.
Also there's a [useAutocomplete hook](https://mui.com/components/autocomplete/#useautocomplete) if we want to not import any styles

Reply:
I'll give MUI Autocomplete a look. I figured importing SemanticUI's CSS would just be a temporary solution so I could continue moving forward on developing this feature for now

Reply:
You can also start with a simple select, and we add a searchable one after